pom.xml

```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <version.eclipse.microprofile.config>1.4</version.eclipse.microprofile.config>
-        <version.eclipse.microprofile.graphql>1.0.2</version.eclipse.microprofile.graphql>
+        <version.eclipse.microprofile.graphql>1.1-SNAPSHOT</version.eclipse.microprofile.graphql>
```

Comment:
We can not build (in CD/CI) against snapshot

Reply:
I know... one of the reasons for a draft PR :-)

Just went back to 1.0.2, but please remember to go to the 1.1-SNAPSHOT from https://github.com/eclipse/microprofile-graphql/pull/266 before testing